paddle/fluid/platform/profiler.cc

```diff
@@ -42,7 +42,9 @@ DEFINE_bool(enable_host_event_recorder_hook,
             false,
             "enable HostEventRecorder, hook Profiler");
 
-DEFINE_bool(enable_record_input_shape, false, "enable input shape recorder");
+DEFINE_bool(enable_record_op_info,
```

Comment:
Please confirm this flag is not public to users so that you can change the flag name.

Reply:
yes, this flag is internal.